src/Futhark/Actions.hs

```diff
@@ -154,15 +154,15 @@ runCC cpath outpath cflags_def ldflags = do
     Right (ExitSuccess, _, _) ->
       return ()
 
-runEMCC :: String -> String -> [String] -> [String] -> [String] -> Bool -> Bool -> FutharkM ()
-runEMCC cpath outpath cflags_def ldflags expfuns rts lib = do
+runEMCC :: String -> String -> FilePath -> [String] -> [String] -> [String] -> Bool -> Bool -> FutharkM ()
+runEMCC cpath outpath classpath cflags_def ldflags expfuns rts lib = do
   ret <-
     liftIO $
       runProgramWithExitCode
         "emcc"
         ( [cpath, "-o", outpath]
             ++ ["-lnodefs.js"] -- "-s", "INITIAL_MEMORY=201326592"] -- ,"-s", "ALLOW_MEMORY_GROWTH=1"]
-            ++ (if rts then ["-s", "--post-js", "futharkClass.js"] else [])
+            ++ (if rts then ["-s", "--post-js", classpath] else [])
```

Comment:
This file `classpath` isn't needed after this statement. Would you recommend deleting it or do you think its helpful to have an unused file that only contains the API laying around?

Reply:
I was wondering about that myself.  In that case I would actually suggest making it a temporary file.  However, the Futhark compiler already leaves intermediate build artifacts in the directory, so it's not crucial to remove it.